types/mongo-sanitize/index.d.ts

```diff
@@ -1,9 +1,9 @@
 // Type definitions for mongo-sanitize 1.0
 // Project: https://github.com/vkarpov15/mongo-sanitize
-// Definitions by: Cedric Cazin <https://github.com/CedricCazin>
+// Definitions by: Cedric Cazin <https://github.com/CedricCazin>, Olga Isakova <https://github.com/penumbra1>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
+// TypeScript Version: 3.3
```

Comment:
what makes 3.3 required instead of 2.3?

Reply:
Yes, sorry I misinterpreted the comment (as the latest tested version as opposed to required). 2.3 is fine.

Reply:
However, --esModuleInterop was introduced in [2.7](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-7.html), and this is when namespace imports started throwing "no call signature".  Does this mean 2.7 a requirement? 
The CI check fails for the same reason and will pass with 2.7. I've rolled back to 2.3 for now, but please let me know if I should change the version to 2.7.